### PR TITLE
add recipe for cantools

### PR DESCRIPTION
**Pull request by danielhrisca**

### CODE REVIEW

File: recipes/cantools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  entry_points:
+    - cantools=cantools.__init__:_main
+
+requirements:
+  build:
+    - python
+    - pip
+    - bitstruct>=3.7.0
+    - pyparsing>=2.0.3
+    - python-can>=2.1.0
```
Comment (CJ-Wright) [inline recipes/cantools/meta.yaml]: Are all of these build deps?

Comment (CJ-Wright) [reply recipes/cantools/meta.yaml]: Also it seems that we don't have python-can in conda-forge, would you be interested in adding it?

**Comment by danielhrisca** [comment]
Hello Christopher,,

I wanted to add python-can yesterday but I didn't get the time.

I will do this as soon as possible.

Thank you.

**Comment by danielhrisca** [comment]
I've created PR #5827 but I'm stuck with the CI not beeing able to import _mock_ since the conda forge packages does not support linux-32 https://anaconda.org/conda-forge/mock

**Pull request merged**